playbooks/add_compute.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Why we need this as separate file. Cannot we just add tag and use related logic in deploy_airship flow

Reply:
Yeah but this is cleaner way of doing because that playbook does other tasks.